docs/tutorials/mastering.md

```diff
@@ -1,60 +1,633 @@
 
-```{image} ../images/sdcore_networking.png
-:alt: Network diagram
-:align: center
-```
-
-To complete this tutorial, you will need four machines (virtual or physical), with access to the networks as follows.
+## 1. Prepare the Host machine
 
-| Machine | CPUs | RAM | Disk | Networks |
-| ------- | ---- | --- | ---- | -------- |
-| Control Plane Kubernetes Cluster     | 4 | 8g | 40g | `management` |
-| User Plane Kubernetes Cluster        | 4 | 4g | 20g | `management`, `access`, `core` |
-| Juju Controller + Kubernetes Cluster | 2 | 8g | 40g | `management` |
-| gNB Simulator Kubernetes  Cluster    | 4 | 4g | 20g | `management`, `ran` |
+A machine running Ubuntu 22.04 with the following resources:
 
-
-## Sample Values
-
-For the purpose of this guide, the following values will be used.
+- At least one NIC with internet access
+- 8 cores
+- 24 GB RAM
+- 150 GiB disk  
 
 ### Networks
 
+The following IP networks will be used to connect and isolate the network functions.
+
 | Name | Subnet | Gateway IP |
 | ---- | ------ | ---------- |
 | `management` | 10.201.0.0/24 | 10.201.0.1 |
 | `access`     | 10.202.0.0/24 | 10.202.0.1 |
 | `core`       | 10.203.0.0/24 | 10.203.0.1 |
 | `ran`        | 10.204.0.0/24 | 10.204.0.1 |
 
+On the host machine, create local network bridges to be used by LXD by adding below configuration under `/etc/netplan/99-sdcore-networks.yaml`.
+
+```console
+cat << EOF | sudo tee /etc/netplan/99-sdcore-networks.yaml
+# /etc/netplan/99-sdcore-networks.yaml
+network:
+  bridges:
+    mgmt-br:
+      addresses:
+        - 10.201.0.14/24
+      routes:
+        - to: default
+          via: 10.201.0.1
+          metric: 110
+    access-br:
+      addresses:
+        - 10.202.0.14/24
+      routes:
+        - to: 10.204.0.0/24
+          via: 10.202.0.1
+    core-br:
+      addresses:
+        - 10.203.0.14/24
+      routes:
+        - to: default
+          via: 10.203.0.1
+          metric: 203
+    ran-br:
+      addresses:
+        - 10.204.0.14/24
+      routes:
+        - to: 10.202.0.0/24
+          via: 10.204.0.1
+
+  version: 2
+EOF
+```
+
+Apply the network configuration.
+
+```console
+sudo netplan apply
+```
+
+### Install and Configure LXD
+
+Install LXD:
+
+```console
+sudo snap install lxd
+```
+
+```console
+cat << EOF | sudo tee ~/preseed.yaml
+config: {}
+networks: []
+storage_pools:
+- config:
+    size: 150GiB
+    source: /var/snap/lxd/common/lxd/disks/default.img
+    zfs.pool_name: default
+  description: ""
+  name: default
+  driver: zfs
+profiles:
+- config: {}
+  description: Default LXD profile
+  devices:
+    root:
+      path: /
+      pool: default
+      type: disk
+  name: default
+projects:
+- config:
+    features.images: "true"
+    features.networks: "true"
+    features.networks.zones: "true"
+    features.profiles: "true"
+    features.storage.buckets: "true"
```

Comment:
After changing project name back to `default` (see the comment above), on a fresh host, I'm getting:
```bash
ubuntu@ip-172-31-35-202:~$ lxd init --preseed < ~/preseed.yaml 
Error: Failed to update local member project "default": Invalid project configuration key "features.storage.buckets"
```

Reply:
I still don't think we wanna use the default project. I think we need to figure out how to handle project creation nicely.

Reply:
Custom LXD project and storage pool creation is added.